lms/djangoapps/instructor/views/api.py

```diff
@@ -3044,10 +3044,8 @@ def generate_certificate_exceptions(request, course_id, generate_for=None):
 
     if generate_for == 'all':
         # Generate Certificates for all white listed students
-        students = User.objects.filter(
-            certificatewhitelist__course_id=course_key,
-            certificatewhitelist__whitelist=True
-        )
+        students = 'all_whitelisted'
+
     elif not all(users):
```

Comment:
I don't think we need this information anymore, since we don't care about the list of students coming from the browser